distribution/src/resources/config-tool/infer.json

```diff
@@ -1 +1,73 @@
-{}
+{
+  "transport.vfs.listener.enable": {
+    "true": {
+      "transport.vfs.listener.class": "org.apache.synapse.transport.vfs.VFSTransportListener"
+    }
+  },
+  "transport.fix.listener.enable": {
+    "true": {
+      "transport.fix.listener.class": "org.apache.synapse.transport.fix.FIXTransportListener"
+    }
+  },
+  "transport.hl7.listener.enable": {
+    "true": {
+      "transport.hl7.listener.class": "org.wso2.carbon.business.messaging.hl7.transport.HL7TransportListener"
+    }
+  },
+  "transport.idoc.listener.enable": {
```

Comment:
for BAPI and idoc shall we have `transport.sap.listener.enable` by default? In addition, we'll have a `transport.sap.listener.bapi.enable` property for BAPI listener and a  `transport.sap.listener.idoc.enable` for idoc. 
When we say `transport.sap.listener.enable = true` idoc listener will get enabled by default. But the BAPI listener will be disabled. If someone wants to enable BAPI listener they will have to explicitly set  `transport.sap.listener.bapi.enable` to true.

Reply:
Same should apply for the sender as well.

Reply:
Fixed